### PR TITLE
PROGRAMMES-5982 - BBC Radio Cymru 2

### PR DESCRIPTION
**Pull request by felipeparaujo**

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/DsShared/Helpers/LiveBroadcastHelper.php [hunk 1]
```diff
@@ -63,6 +63,7 @@ class LiveBroadcastHelper
         'bbc_radio_foyle'                       => ['network', ['networkUrlKey' => 'radiofoyle']],
         'bbc_radio_wales_fm'                    => ['network', ['networkUrlKey' => 'radiowales']],
         'bbc_radio_cymru'                       => ['network', ['networkUrlKey' => 'radiocymru']],
+        'bbc_radio_cymru_2'                     => ['network', ['networkUrlKey' => 'radiocymru2']],
```
Comment (BjornTwachtmann) [inline src/DsShared/Helpers/LiveBroadcastHelper.php]: This is a 404 on live. Are we sure it's going to be created?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

**Pull request merged**